demos/gallery/samples/widget/ChartSeries.js

```diff
@@ -0,0 +1,16 @@
+import { ChartSeries } from "@hpcc-js/composite";
+
+new ChartSeries()
+    .target("target")
+    .chartType("Area")
+    .prependMissingData(true)
+    .titleSuffix(" - Burglary rate per 100,000")
+    .data([
+        ["Aventura",919],["Aventura",601],["Aventura",478],["Aventura",578],["Aventura",507],["Aventura",548],["Aventura",451],["Aventura",499],["Aventura",421],["Aventura",370],["Aventura",353],["Aventura",464],["Aventura",327],["Aventura",262],["Aventura",221],["Aventura",203],["Aventura",210],
+        ["Miami Lakes",750],["Miami Lakes",730],["Miami Lakes",755],["Miami Lakes",642],["Miami Lakes",564],["Miami Lakes",537],["Miami Lakes",588],["Miami Lakes",647],["Miami Lakes",412],["Miami Lakes",312],["Miami Lakes",295],["Miami Lakes",280],["Miami Lakes",307],
+        ["Seminole",633],["Seminole",904],["Seminole",1062],["Seminole",665],["Seminole",447],["Seminole",429],["Seminole",488],["Seminole",574],["Seminole",688],["Seminole",504],["Seminole",464],["Seminole",606],["Seminole",394],["Seminole",277],["Seminole",322],
+        ["Wellington",887],["Wellington",683],["Wellington",969],["Wellington",758],["Wellington",649],["Wellington",699],["Wellington",637],["Wellington",513],["Wellington",547],["Wellington",618],["Wellington",705],["Wellington",532],["Wellington",467],["Wellington",403],["Wellington",438],["Wellington",353],
+        ["Weston",491],["Weston",273],["Weston",174],["Weston",227],["Weston",119],["Weston",193],["Weston",217],["Weston",235],["Weston",194],["Weston",249],["Weston",209],["Weston",206],["Weston",179],["Weston",219],["Weston",114],["Weston",89],["Weston",459]
```

Comment:
Data shape looks weird - You should be able to use our standard data shape:
```
    .data([
        ["Geography", 75, 68, 65],
        ["English", 45, 55, -52],
        ["Math", 98, 92, 90],
        ["Science", 66, 60, 72]
    ])
```
Where Year 1, 2 and 3 become the different chart instances.

Reply:
Okay, but I'd like for it to be able to detect this current data shape and automagically transform it into our current ND standard. Is that alright with you?

Reason being... this data shape is really common and we currently don't have a way to visualize it without manually transforming it into our standard ND shape.

Reply:
Data transformation helpers would be a great addition to the "util" module?

Reply:
> Data transformation helpers would be a great addition to the "util" module?

I suppose a transformation helper would need a classifier first? Or do you think they'd need to be manually selected by the user?

Reply:
Our widgets should be consistent with the default data shape they take as input - if there are other "popular" shapes, we could provide transformers to our format - but I would like to avoid having any one widget taking a different shape,

Reply:
Yep, I'm working on it